sdk/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -0,0 +1,164 @@
+    }
+
+    /**
+     * Deactivates an actor for an actor type with given actor id.
+     * @param actorTypeName Actor type name to deactivate the actor for.
+     * @param actorId Actor id for the actor to be deactivated.
+     */
+    static void Deactivate(String actorTypeName, String actorId)
+    {
+        // uncomment when ActorManager implemented
+        // return instance.GetActorManager(actorTypeName).DeactivateActor(new ActorId(actorId));
+    }
+
+    /**
+     *  Invokes the specified method for the actor, this is mainly used for cross language invocation.
+     * @param actorTypeName Actor type name to invoke the method for.
+     * @param actorId Actor id for the actor for which method will be invoked.
+     * @param actorMethodName Method name on actor type which will be invoked.
+     * @param requestBodyStream Payload for the actor method.
+     * @param responseBodyStream Response for the actor method.
+     * @return
+     */
+    static void Dispatch(String actorTypeName, String actorId, String actorMethodName, byte[] requestBodyStream, byte[] responseBodyStream)
+    {
+        // uncomment when ActorManager implemented
+        // return instance.GetActorManager(actorTypeName).Dispatch(new ActorId(actorId), actorMethodName, requestBodyStream, responseBodyStream);
+    }
+
+    /**
+     * Fires a reminder for the Actor.
+     * @param actorTypeName Actor type name to invoke the method for.
+     * @param actorId Actor id for the actor for which method will be invoked.
+     * @param reminderName The name of reminder provided during registration.
+     * @param requestBodyStream Payload for the actor method
+     */
+    static void FireReminder(String actorTypeName, String actorId, String reminderName, byte[] requestBodyStream)
+    {
+        // uncomment when ActorManager implemented
+        // return instance.GetActorManager(actorTypeName).FireReminder(new ActorId(actorId), reminderName, requestBodyStream);
+    }
+
+    /**
+     * Fires a timer for the Actor.
+     * @param actorTypeName Actor type name to invoke the method for.
+     * @param actorId Actor id for the actor for which method will be invoked.
+     * @param timerName The name of timer provided during registration.
+     */
+    static void FireTimer(String actorTypeName, String actorId, String timerName)
+    {
+        // uncomment when ActorManager implemented
+        // return instance.GetActorManager(actorTypeName).FireTimerAsync(new ActorId(actorId), timerName);
+    }
+
+    private ActorManager GetActorManager(String actorTypeName) throws IllegalStateException
+    {
+        ActorManager actorManager = this.actorManagers.get(actorTypeName);
+
+        if (actorManager == null)
+        {
+            String errorMsg = String.format("Actor type %s is not registered with Actor runtime.", actorTypeName);
+
+            // TODO - figure out logging.  For now just print it straight out
```

Comment:
For logging, add an interface and use it, printing to console can be one implement ation of it.